### PR TITLE
MAINT: Drop more FreeSurfer components

### PR DESCRIPTION
**Pull request by effigies**
I think for testing niworkflows we only need `bin/` and possibly some `lib/`. Letting Circle decide if I'm right.

Meant to post this against Oscar's branch, but that got merged.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,23 +25,17 @@ RUN apt-get update && \
...
-    --exclude='freesurfer/subjects/fsaverage4' \
-    --exclude='freesurfer/subjects/cvs_avg35' \
-    --exclude='freesurfer/subjects/cvs_avg35_inMNI152' \
-    --exclude='freesurfer/subjects/bert' \
-    --exclude='freesurfer/subjects/lh.EC_average' \
-    --exclude='freesurfer/subjects/rh.EC_average' \
-    --exclude='freesurfer/subjects/sample-*.mgz' \
-    --exclude='freesurfer/subjects/V1_average' \
+    --exclude='freesurfer/mni' \
+    --exclude='freesurfer/subjects' \
```
Comment (oesteban) [inline Dockerfile]: you can't remove `freesurfer/subjects` in full, `fsaverage{5,6,}` have to be there

Comment (effigies) [reply Dockerfile]: For niworkflows?

Comment (oesteban) [reply Dockerfile]: In principle not, maybe you can get away with `mkdir -p /opt/freesurfer/subjects`.

Comment (effigies) [reply Dockerfile]: Oh, I see. The failing test.

**Comment by effigies** [comment]
Ah thanks for updating this.

**Comment by effigies** [comment]
Oh, looks like we need `fsaverage`, still.

**Comment by effigies** [comment]
Eh. This is more trouble than it's worth for a minor potential pickup in speed.

**Pull request closed without merging**